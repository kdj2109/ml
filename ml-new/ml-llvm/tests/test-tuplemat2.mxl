int main() { 
	float[3][2:2] x;
	float b;
	float[3] o;
	float o1; 
	float o2; 
	float[3] c;
	float[3] i; 
	x=[|(11.11,22.22,33.33),(55.55,66.66,77.77)|(17.17,38.38,17.18),(17.37,38.55,35.35)|];
        x[0:0]= [45.5, 54.2, 33.33];
        c = x[0:0];
        b= c[0];
        printf(b);
	b = c[1];
	printf(b);
	o = x[1:0];
	o1 = o[0];
	o2 = o[1];
	o[0] = o1;
	o[1] = o2;
	o[2] = 22.3;
	x[1:0] = o; 
	c = x[1:0];
	b = c[2];
	
	printf(b);
	return 0; 
}
