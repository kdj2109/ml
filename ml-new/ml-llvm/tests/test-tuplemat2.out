45.500000
54.200000
22.300000
